extensions/amp-iframe/0.1/amp-iframe.js

```diff
@@ -618,6 +619,9 @@ export class AmpIframe extends AMP.BaseElement {
         );
       }
     }
+    if (this.iframe_) {
+      this.propagateAttributes(['title'], this.iframe_);
```

Comment:
i would ideally propagate all `ATTRIBUTES_TO_PROPAGATE`, but this was causing an [e2e test error](https://travis-ci.org/ampproject/amphtml/jobs/657440421) when I tried that, so only propagating `title` on mutate.

Reply:
A comment here indicating why only `title` is propagated would be nice.